src/GraphQL.Tests/Serialization/SystemTextJson/ObjectDictionaryConverterTests.cs

```diff
@@ -1,297 +0,0 @@
-using System.Collections.Generic;
-using System.Text.Encodings.Web;
-using System.Text.Json;
-using GraphQL.SystemTextJson;
-using Shouldly;
-using Xunit;
-
-namespace GraphQL.Tests.Serialization.SystemTextJson
-{
-    public class ObjectDictionaryConverterTests
```

Comment:
Do we have tests for `InputsConverter`? InputsConverterTests, right?

Reply:
Yes